dev/com.ibm.ws.grpc_fat/fat/src/com/ibm/ws/fat/grpc/FATSuite.java

```diff
@@ -16,7 +16,6 @@
 
 @RunWith(Suite.class)
 @SuiteClasses({
-                ClientConfigTests.class,
```

Comment:
Was this file/update accidentally checked in? It doesn't seem to have anything to do with JPA.

Reply:
Yeah I don't know where that crept in.  Fixed.